pkg/services/searchV2/index_test.go

```diff
@@ -429,8 +429,8 @@ var dashboardsWithFolders = []dashboard{
 		summary: &models.EntitySummary{
 			Name: "Dashboard in folder 1",
 			Nested: []*models.EntitySummary{
-				newNestedPanel(1, "Panel 1"),
-				newNestedPanel(2, "Panel 2"),
+				newNestedPanel(1, 2, "Panel 1"),
```

Comment:
These test changes are necessary, as location for panels needs to be of the correct format (folderID/dashID) for the access control filter checks to pass. So I had to create general folder for dashboards without a folder and change the way that UID for nested panels is set.